dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -503,6 +504,30 @@ public void testUpdateWorkflowPath() throws ApiException {
         assertEquals("test workflow path should be the same as default workflow path, it is " + testpath, "/Dockstore.cwl", testpath);
     }
 
+    @Test
+    public void testAddingWorkflowForumUrl() throws ApiException {
+        // Set up webservice
+        ApiClient webClient = WorkflowIT.getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(webClient);
+
+        UsersApi usersApi = new UsersApi(webClient);
+        usersApi.getUser();
+
+        Workflow workflow = workflowsApi
+                .manualRegister("github", "DockstoreTestUser2/test_lastmodified", "/Dockstore.cwl", "test-update-workflow", "cwl",
+                        "/test.json");
+
+        //update the forumUrl to hello.com
+        workflow.setForumUrl("hello.com");
+        workflowsApi.updateWorkflow(workflow.getId(), workflow);
+        workflowsApi.refresh(workflow.getId());
```

Comment:
Do you need this? Shouldn't the forumUrl already be in the DB after line 522? Or are you trying to verify that it sticks after refresh?

Reply:
Hm I included the refresh because I was modelling the structure of other update tests. I think verifying it still sticks after is refresh is a good idea, mimics the behaviour of when a user updates the forumUrl